js/main.js

```diff
@@ -130,3 +139,50 @@ function handleSnooze(event) {
 function scrollToBottom(element) {
   element.scrollTop = element.scrollHeight;
 }
+
+function handleCollection(event) {
+  $backToJokesButton.className = 'back-to-jokes-button';
+  $moreJokesButton.className = 'hidden';
+  $collectionButton.className = 'hidden';
+  $bigLaughingEmoji.className = 'big-laughing-emoji justify-center';
+  while ($jokeTarget.lastChild) {
+    $jokeTarget.removeChild($jokeTarget.lastChild);
+  }
```

Comment:
You're doing this in the other function you wrote too, you should make this its own named function like `emptyElementContents` that receives an argument of the element to delete the contents of.